scripts/scil_compute_freewater.py

```diff
@@ -0,0 +1,203 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Compute Free Water maps [1] using AMICO.
+"""
+
+import argparse
+import os
+import shutil
+import tempfile
+
+import amico
+import numpy as np
+
+from scilpy.io.utils import (add_overwrite_arg, add_processes_arg,
+                             assert_inputs_exist, assert_outputs_exist)
+
+amico.core.setup()
+
+EPILOG = """
+Reference:
+    [1] Pasternak 0, Sochen N, Gur Y, Intrator N, Assaf Y.
+        Free water elimination and mapping from diffusion mri.
+        Magn Reson Med. 62 (3) (2009) 717-730.
+"""
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        epilog=EPILOG)
+
+    p.add_argument('in_dwi',
+                   help='DWI file.')
+
+    p.add_argument('--mask',
+                   help='Mask filename.')
+
+    g1 = p.add_argument_group('Gradients / scheme')
+    g1.add_argument('--bval',
+                    help='Bval filename, in FSL format.')
+    g1.add_argument('--bvec',
+                    help='Bvec filename, in FSL format.')
+    g1.add_argument('--scheme_file',
+                    help='AMICO scheme file, '
+                         'can replace --bval/--bvec.')
+    g1.add_argument('--bstep', type=int, nargs='+',
+                    help='List of unique bvals in your data. It prevents '
+                         ' errors when each bval is around the actual bval.')
+
+    p.add_argument('--para_diff', type=float, default=1.5e-3,
+                   help='Axial diffusivity (AD) in the CC. [%(default)s]')
+    p.add_argument('--iso_diff', type=float, default=3e-3,
+                   help='Mean diffusivity (MD) in ventricles. [%(default)s]')
+    p.add_argument('--perp_diff_min', type=float, default=0.1e-3,
+                   help='Radial diffusivity (RD) minimum. [%(default)s]')
+    p.add_argument('--perp_diff_max', type=float, default=0.7e-3,
+                   help='Radial diffusivity (RD) maximum. [%(default)s]')
+
+    p.add_argument('--lambda1', type=float, default=0.0,
+                   help='First regularization parameter. [%(default)s]')
+    p.add_argument('--lambda2', type=float, default=1e-3,
+                   help='Second regularization parameter. [%(default)s]')
+
+    p.add_argument('--mouse', action='store_true',
+                   help='If set, use mouse fitting profile.')
+
+    p.add_argument('--output_dir',
```

Comment:
You should put a default, such as `results/`
If the user wants it in the current folder they should provide `./`